addons/hr_expense/models/hr_expense.py

```diff
@@ -389,40 +428,80 @@ def action_view_sheet(self):
     def _get_default_expense_sheet_values(self):
         if any(expense.state != 'draft' or expense.sheet_id for expense in self):
             raise UserError(_("You cannot report twice the same line!"))
+        if all(expense.total_amount_company == 0.0 for expense in self):
+            raise UserError(_("You cannot report the expense with 0 total!"))
         if len(self.mapped('employee_id')) != 1:
             raise UserError(_("You cannot report expenses for different employees in the same report."))
         if any(not expense.product_id for expense in self):
             raise UserError(_("You can not create report without category."))
 
-        todo = self.filtered(lambda x: x.payment_mode=='own_account') or self.filtered(lambda x: x.payment_mode=='company_account')
-        if len(todo) == 1:
-            expense_name = todo.name
+        # If there is an expense with total_amount_company == 0, it means that expense has not been processed by OCR yet
+        expenses_with_amount = self.filtered(lambda x: x.total_amount_company != 0)
```

Comment:
```suggestion
        expenses_with_amount = self.filtered('total_amount_company')
```
unless `total_amount_company` can be negative? :thinking:

Reply:
In line with some other comments, I uses here as well - float_compare